lib/src/components/NxTextInput/__tests__/NxTextInput.test.tsx

```diff
@@ -16,18 +16,6 @@ describe('NxTextInput', function() {
       },
       getShallowComponent = enzymeUtils.getShallowComponent<Props>(NxTextInput, minimalProps);
 
-  it('renders a text input wrapped in a NxTooltip by default', function() {
```

Comment:
The non-tooltip aspects of these two tests need to be restored

Reply:
> The non-tooltip aspects of these two tests need to be restored

I think I restored them correctly here: https://github.com/sonatype/sonatype-react-shared-components/pull/57/commits/4f062184bfa8471193a4e5b93197713ba09b58ba